src/Servers/Kestrel/samples/Http3SampleApp/Program.cs

```diff
@@ -25,32 +27,68 @@ public static void Main(string[] args)
                 webHost.UseKestrel()
                 .ConfigureKestrel((context, options) =>
                 {
-                    var cert = CertificateLoader.LoadFromStoreCert("localhost", StoreName.My.ToString(), StoreLocation.CurrentUser, false);
+                    // CERTIFICATE STUFF THAT SHOULD BE MERGED WITH KESTREL EVENTUALLY///////////
+                    X509Certificate2 cert = null;
+                    var store = new X509Store("testing2", StoreLocation.CurrentUser);
```

Comment:
nit: I know this is just a sample app but I'd use a more meaningful name here so people know where it came from if they see it in their store.